Minimal change required to show all validation errors for all required fields

macie-korte opened the pull request:
Minimal change required to show all validation errors for all required fields instead of only a small subset.

I know you will probably not want this change as-is, I am opening this PR only to demonstrate what change was needed to make the desired behavior work for me.

miki725 commented on drf_braces/serializers/form_serializer.py:
```diff
@@ -41,9 +41,7 @@ def run_validation(self, data):
-            if any([not self.parent.partial,
-                    self.parent.Meta.failure_mode == FormSerializerFailure.fail,
-                    self.field_name in self.parent.Meta.minimum_required]):
+            if self.field_name in self.parent.Meta.minimum_required:
```
how does this help? the original conditions are more strict when validating fields

miki725 replied:
now it makes sense. you want to bypass the DRF field validations in order to force form validations to kick in. better place for that would be to add an option in the `Meta` which will force the serializer to not include any DRF fields and will force validation to happen in the form.

places to look in the code:

`FormSerializerBase .get_fields`
`FormSerializerOptions`

macie-korte commented:
FYI: I have deleted many required fields from the applicant JSON and ran validation.

BEFORE this change, I got the 400 result:

    {
      "applicant_form": {
        "line_1_address": [
          "This field may not be blank."
        ],
        "city": [
          "This field may not be blank."
        ],
        "zip_code": [
          "This field may not be blank."
        ]
    },

Which would not change no matter how many fields I set to blank/null values.

AFTER this change, I got the 400 result:

    {
      "applicant_form": {
        "line_1_address": [
          "This field is required."
        ],
        "marital_status_code": [
          "This field is required."
        ],
        "city": [
          "This field is required."
        ],
        "mortgage_payment_or_rent": [
          "This field is required."
        ],
        "birth_date": [
          "This field is required."
        ],
        "employment_status_code": [
          "This field is required."
        ],
        "zip_code": [
          "This field is required."
        ]
      }
    }

which is a full list based on all I deleted for this request.
